src/features/commands.ts

```diff
@@ -33,11 +34,18 @@ export default function registerCommands(server: OmniSharpServer, reporter: Tele
     let attachItemsProvider = DotNetAttachItemsProviderFactory.Get();
     let attacher = new AttachPicker(attachItemsProvider);
     let d6 = vscode.commands.registerCommand('csharp.listProcess', () => attacher.ShowAttachEntries());
+
     // Register command for generating tasks.json and launch.json assets.
     let d7 = vscode.commands.registerCommand('dotnet.generateAssets', () => generateAssets(server));
+
+    // Register command for remote process picker for attach
     let d8 = vscode.commands.registerCommand('csharp.listRemoteProcess', (args) => RemoteAttachPicker.ShowAttachEntries(args));
 
-    return vscode.Disposable.from(d1, d2, d3, d4, d5, d6, d7, d8);
+    // Register command for adapter executable command.
+    let d9 = vscode.commands.registerCommand('csharp.coreclrAdapterExecutableCommand', (args) => registerAdapterExecutionCommand(channel).then(adapterExecutableCommand => adapterExecutableCommand));
+    let d10 = vscode.commands.registerCommand('csharp.clrAdapterExecutableCommand', (args) => registerAdapterExecutionCommand(channel).then(adapterExecutableCommand => adapterExecutableCommand));
```

Comment:
Does the `.then(adapterExecutableCommand => adapterExecutableCommand)` actually do something? It seems like it is equivalent to just setting the command to registerAdapterExecutionCommand.

Reply:
Nope, I'm just terrible at Typescript.